packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -24,90 +24,48 @@ This guide explains how to use Okta as the user store for your server-side web a
 * Knowledge of building server-side Web applications
 * A project or application to add authentication to.
 
-**Sample code**
-
-<StackSelector snippet="samplecode" noSelector/>
-
 ---
 
-## Refresh tokens and web apps
-
-With browser-based apps, the risk of the refresh token being compromised is high when a persistent refresh token is used. This threat is greatly reduced by rotating refresh tokens. [Refresh token rotation](/docs/guides/refresh-tokens/main/#refresh-token-rotation) helps a public client to securely rotate refresh tokens after each use. A new refresh token is returned each time the client makes a request to exchange a refresh token for a new access token. Refresh token rotation works with SPAs, native apps, and web apps in Okta.
-
-See the [OAuth 2.0 for Browser-Based Apps specification](https://tools.ietf.org/html/draft-ietf-oauth-browser-based-apps-05#page-10) for the latest spec information on using refresh tokens with browser-based apps.
-
-## Understand the callback route
-
-To sign users in, your application redirects the browser to an Okta-hosted sign-in page. Okta then redirects back to your application with information about the user. You can learn more about how this works on [Okta-hosted flows](/docs/concepts/okta-hosted-flows/).
-
-Your web application must host a route that Okta sends information to when a user signs in. This route is called a callback route or redirect URI.
-
-The callback route is not seen by the user, and it's not the user's final destination. It's just one step in the authentication redirect flow.
-
-<StackSelector snippet="define-route" noSelector/>
-
-Our examples use `/authorization-code/callback` as a default route path, but you can change this. The route path is used in the next step.
-
 ## Create an Okta app integration
 
-Before you can sign a user in, you need to create an Okta app integration that represents your web application.
+Before you begin, you'll need a free Okta developer account. Install the [Okta CLI](https://cli.okta.com/) and run `okta register` to sign up for a new account. If you already have an account, run `okta login`. Then, run `okta apps create web`. Use "Quickstart" when prompted for a name.
+
+Accept the default Redirect URI values provided for you. That is, a Login Redirect of `http://localhost:8080/login/oauth2/code/okta` and a Logout Redirect of `http://localhost:8080`.
 
-1. Sign in to your Okta organization with your administrator account.
+<!--
```

Comment:
I think there should be instructions for the Admin Console, but they should be at the bottom or in a collapsable section since it's so much longer than the CLI instructions.